qa/rpc-tests/feature_fedpeg.py

```diff
@@ -47,9 +49,30 @@ def __init__(self):
         self.setup_clean_chain = True
         self.num_nodes = 4
 
-    def setup_network(self, split=False):
+    def add_options(self, parser):
+        parser.add_option("--bitcoinddir", dest="bitcoinddir", default="",
```

Comment:
perhaps use --binpath or something directly instead of appending bitcoind in the test?
That way we could also use another elements binary, perhaps an older one to test compatibility or a newer one for testing rebases.
For that we would need to decouple another option --parent_is_bitcoinlike or something of the sort.

Reply:
Changed to `--parent_bitcoin` and `--parent_binpath=` :)